src/pages/index.js

```diff
@@ -335,11 +362,105 @@ const IndexPage = (props) => {
                     {contributorCards}
                 </Box>
             </Box>
+            <Box
+                sx={{
+                    padding: theme.spacing(5, 0),
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                }}
+            >
+                <Box
+                    sx={{
+                        padding: theme.spacing(3),
+                        borderRadius: 5,
+                        maxWidth: 'fit-content',
+                        height: 'fit-content',
+                        backgroundColor: 'rgb(218, 209, 198, 0.3)',
+                        display: 'flex',
+                        justifyContent: 'center',
+                        alignItems: 'center',
+                    }}
+                >
+                    <Stack
+                        direction='row'
+                        gap={3}
+                        justifyItems='center'
+                        alignItems='center'
+                    >
+                        <Box
+                            sx={{
+                                display: 'flex',
+                                justifyContent: 'center',
+                                alignItems: 'center',
+                            }}
+                        >
+                            <img
+                                src={thankYou[6]?.replace(/['"]+/g, '')}
+                                alt='Random contributor image'
+                                width={isDesktop ? 100 : isMobile ? 50 : 90}
+                                height={isDesktop ? 100 : isMobile ? 50 : 90}
+                                style={{
+                                    marginTop: 'auto',
+                                    marginBottom: 'auto',
+                                }}
+                            />
+                        </Box>
+                        <Box
+                            sx={{
+                                fontSize: isMobile ? 'small' : 'medium',
+                            }}
+                        >
+                            Thank you{' '}
+                            {thankYou.filter((item) => item?.trim() === '')
+                                .length === 0 && (
+                                <a
+                                    target='_blank'
+                                    href={thankYou[5]?.replace(/['"]+/g, '')}
+                                >
+                                    {thankYou[3] === true
+                                        ? thankYou[3]?.replace(/['"]+/g, '')
+                                        : thankYou[2]?.replace(/['"]+/g, '')}
+                                </a>
+                            )}
+                            <br />
+                            for making {thankYou[7]?.replace(/['"]+/g, '')} pull
+                            requests(s)
```

Comment:
```suggestion
                            request{thankYou[7]?.replace(/['"]+/g, '') == 1
                                ? ''
                                : 's'
```
Pluralize `pull request` only if there is more than one.

Reply:
Done